cpp/src/arrow/compute/kernels/test_util.h

```diff
@@ -97,5 +99,63 @@ using TestingStringTypes =
 
 static constexpr random::SeedType kRandomSeed = 0x0ff1ce;
 
+struct ScalarFunctionPropertyTestParam {
```

Comment:
I'm not sure what the name "property" is supposed to refer to.

Reply:
https://en.wikipedia.org/wiki/Property_testing